packages/swapper/src/swappers/ThorchainSwapper/evm/utils/getThorTxData.ts

```diff
@@ -33,8 +34,8 @@ export const getThorTxInfo = async ({
   if (maybeInboundAddress.isErr()) throw maybeInboundAddress.unwrapErr()
   const inboundAddress = maybeInboundAddress.unwrap()
 
-  const router = inboundAddress.router
-  const vault = inboundAddress.address
+  const router = inboundAddress.router as Address
+  const vault = inboundAddress.address as Address
```

Comment:
```suggestion
  const router = getAddress(inboundAddress.router)
  const vault = getAddress(inboundAddress.address)
```

Reply:
still needed a cast to `string`, but done :)

Reply:
https://github.com/shapeshift/web/pull/7687/commits/a25d4846d80b0a654ecd7d2115014b963466fc17 LGTM!